api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -221,41 +221,37 @@ public void sendBetaAccessCompleteEmail(final String contactEmail, final String
         .build();
   }
 
+  private String getFreeCreditsResolutionText() {
+    if (workbenchConfigProvider.get().featureFlags.enableBillingUpgrade) {
+      return "you can request additional free credits by contacting support "
```

Comment:
this is not a whole sentence because the clause before this differs.

Reply:
thank you for preemptively acknowledging my grammar neuroses